scripts/ensure_exports_are_documented.py

```diff
@@ -52,10 +84,18 @@ def get_names_from_all_list() -> list[str]:
 
 def ensure_exports_are_documented() -> bool:
     success = True
-    for name in get_names_from_all_list():
-        if not is_documented(name):
-            print(f"{name} is not documented")
-            success = False
+    used_deprecations = set()
+    for package_dir in ("globus_sdk", "globus_sdk/scopes"):
```

Comment:
nit:

We'll probably want to include more than just sdk & scopes, maybe pull this tuple into a static-defined constant to make it more clear where we would add those?

Reply:
Yeah, I can put it at top of script. I definitely want to grow this list.